execute_process_mass_data.py

```diff
@@ -49,7 +49,7 @@
 # plot_binned_pmf_array(gases, list_of_arrays, create_bins_results, bin_compositions_results)
```

Comment:
generally the word "squash" in a commit message indicates that you forgot to actually squash